pandas/core/frame.py

```diff
@@ -2432,7 +2432,8 @@ def eval(self, expr, inplace=False, **kwargs):
         return _eval(expr, inplace=inplace, **kwargs)
 
     def select_dtypes(self, include=None, exclude=None):
-        """Return a subset of a DataFrame including/excluding columns based on
+        """
+        Return a subset of a DataFrame including/excluding columns based on
```

Comment:
Let's shorten this a bit so that it fits on a line. How about

    Return a subset of DataFrame based on the column dtypes.

Reply:
Thanks for your comments. I've got one question about docstring guideline. 

I looked through several comments left on other PR and found this type of docstring is correct.

```
"""This type of docstring.
```

I ran the script mentioned in the guideline but it said docstring should be start with new line like this.

```
"""
This type of docstring.
```

Which one is correct one?

Thanks.

Reply:
It's the second one, so on new line (we changed that in the docstring guie rather late before the sprint)

Reply:
@jorisvandenbossche Thanks. I've made docstrings based on the second one and changed things you and other contributors pointed out